src/__tests__/macro.js

```diff
@@ -4,14 +4,14 @@ import plugin from 'babel-macros'
 import prettier from 'prettier'
 import {prettier as prettierConfig} from 'kcd-scripts/config'
 
-const projectRoot = path.join(__dirname, '../../')
+const projectRoot = path.join(__dirname, '../../').replace(/\\/g, '/')
 
 expect.addSnapshotSerializer({
   print(val) {
     return val
       .split(projectRoot)
       .join('<PROJECT_ROOT>/')
-      .replace(/fixtures/g, 'my-files')
+      .replace(/fixtures/g, 'my_files')
```

Comment:
Why did you change this?

Reply:
I realized later why you did that. I changed it to simply `files` because I don't like snake_case :wink:

Thanks!